### DISCUSSION TITLE
【已结束】Serverless Devs Docs 国际化

### DISCUSSION THREAD

**Discussion by anycodes** [Serverless-Devs/Serverless-Devs#310] [General]
目前 Serverless Devs 的文档是中文文档：https://github.com/Serverless-Devs/Serverless-Devs/tree/master/docs

需要将这一部分文档国际化，欢迎有兴趣的小伙伴一起参与贡献。

贡献参考：https://github.com/Serverless-Devs/Serverless-Devs/blob/master/CONTRIBUTING.md

也可以参考以下流程：
1. `fork`仓库
2. 选择想要翻译的内容
3. 在本`discussion`下面留言，告知大家你要翻译那个部分，并看一下之前是否有人已经翻译了这部分
4. 开始翻译
5. 翻译完成提一个`pr`
6. 等待合并到`master`
7. 等待社区小礼品和签名书籍

---------------

另外一种贡献方法：https://crowdin.com/project/serverless-devs

在`crowdin`平台，参与项目贡献，主要流程是：
1. 在本`discussion`下面留言，告知大家你要翻译那个部分，并看一下之前是否有人已经翻译了这部分
2. 在`crowdin`平台进行翻译
3. 等待翻译内容被合并到Github的`master`
4. 等待社区小礼品和签名书籍

**Comment by anycodes** [comment]
PS：贡献之后，被合并到`master`的还需要提交一下自己的信息到这个页面：https://github.com/Serverless-Devs/Serverless-Devs/blob/master/CONTRIBUTORS.md

**Comment by yankfy** [comment]
翻译 Serverless-Devs/blob/master/docs/zh/install.md

**Reply by anycodes** [reply]
大佬太强了已经看到了pr

**Comment by yankfy** [comment]
翻译 Serverless-Devs/blob/master/docs/zh/quick_start.md

**Comment by llkevin13579** [comment]
能否利用crowdin进行国际化？

**Reply by anycodes** [reply]
感谢大佬支持，现在更新文档国际化的附加方案：https://crowdin.com/project/serverless-devs

**Comment by Fitz-devs** [comment]
翻译 Serverless-Devs/docs/zh/default_provider_config/alibabacloud.md

**Comment by JasonLamv-t** [comment]
目前看似乎都比较完善？现在有哪些部分需要翻译呢？

**Reply by anycodes** [reply]
感谢大佬，目前已经基本完成翻译
